insights/specs/insights_archive.py

```diff
@@ -108,8 +108,7 @@ class InsightsArchiveSpecs(Specs):
     lscpu = simple_file("insights_commands/lscpu")
     lsmod = simple_file("insights_commands/lsmod")
     lsof = simple_file("insights_commands/lsof")
-    lspci = simple_file("insights_commands/lspci")
-    lspci_kernel = simple_file("insights_commands/lspci_-k")
+    lspci = first_file(["insights_commands/lspci_-k", "insights_commands/lspci"])
```

Comment:
Consistent with the commend above this should be a `simple_file`.

Reply:
Done